Assignment2 crud operation

```diff
@@ -48,17 +52,68 @@ int main()
     return 0;
 }
 
+void readInput(User *user)
+{
+    printf("Enter ID: ");
+    while (scanf("%d", &user->id) != 1)
```

Comment:
can you cover scenario to avoid negative integer , ID will be positive value , apply same for reading age too.